app/views/pages/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1>New Page</h1>
+<%= form_for(@page) do |f| %>
+  <% if @page.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@page.errors.count, "error") %> prohibited this page from being saved:</h2>
+
+      <ul>
+      <% @page.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :code %><br>
+    <%= f.text_field :code %>
+  </div>
+  <div class="field">
+    <%= f.label :title %><br>
+    <%= f.text_field :title %>
+  </div>
+  <div class="field">
+    <%= f.label :body %>
+    <%= f.text_area :body, size: '70x30' %>
+  </div>
+  <script src="//cdn.ckeditor.com/4.4.4/standard/ckeditor.js"></script>
```

Comment:
Is it possible to use the [latest and greatest version](http://ckeditor.com/download) of CKEditor?